hack/debug.sh

```diff
@@ -52,6 +52,7 @@ log_system () {
     echo_and_eval  cat /etc/resolv.conf                               &> $logpath/resolv.conf
     echo_and_eval  lsmod                                              &> $logpath/modules
     echo_and_eval  sysctl -a                                          &> $logpath/sysctl
+    echo_and_eval  cat "$config"                                      &> $logpath/master-config.yaml
```

Comment:
This puts the assumption that log_system() is always called from do_master() or do_node()?  and in case of node, master-config.yaml is confusing, it should be node-config.yaml?
Why don't we capture and log this info in do_master() and do_node() itself?